recipes/open62541pp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,32 @@
+#include <open62541pp/open62541pp.h>
+
+#include <iostream>
+
+
+int main() {
+    opcua::Server server(4840 /* port */);
```

Comment:
Let's simplify the test package not to have to use server classes - reads like this could be hitting the network layer :)

Reply:
Yes it does. The example is similar to the open62541 one: https://github.com/conan-io/conan-center-index/blob/master/recipes/open62541/all/test_package/test_package.c

I could provide a test without any server/client connection if needed.

Reply:
That'd be ace yes :) thanks!

Reply:
Ops, I was wrong. The example seems to be fine. The actual server connection is only instantiated if `server.run()` is called.

Reply:
I can simply move the test code to a dummy function that does not get called from main().

Reply:
> The actual server connection is only instantiated if server.run() is called.

Having this as a comment in the code itself then should be enough :)